### ISSUE TITLE
test_sync in 4.45 looks to fail on 32 bit

### ISSUE THREAD

**Issue by jeanconn** [sot/eng_archive#174]
No sure if this is expected (test should be skipped) or a problem (code should be fixed).

```
=================================== FAILURES ===================================
_____________________________ test_sync[pcad13eng] _____________________________

tmpdir = local('/tmp/pytest-of-SOT/pytest-1/test_sync_pcad13eng_0')
content = 'pcad13eng'

    @pytest.mark.parametrize('content', list(CONTENTS))
    def test_sync(tmpdir, content):
>       check_content(tmpdir, content)

Ska/engarchive/tests/test_sync.py:343:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
Ska/engarchive/tests/test_sync.py:314: in check_content
    msids=msids)
Ska/engarchive/tests/test_sync.py:271: in make_stub_content
    make_stub_h5_col(msid, row0, row1, basedir_ref, basedir_stub)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

msid = 'TIME', row0 = 288834752, row1 = 288837952
basedir_ref = PosixPath('/tmp/pytest-of-SOT/pytest-1/test_sync_pcad13eng_0/orig')
basedir_stub = PosixPath('/tmp/pytest-of-SOT/pytest-1/test_sync_pcad13eng_0/test')

    def make_stub_h5_col(msid, row0, row1, basedir_ref, basedir_stub):
        fetch.ft['msid'] = msid

        with set_fetch_basedir(basedir_ref):
            file_ref = fetch.msid_files['data'].abs

        if not Path(file_ref).exists():
            return

        with tables.open_file(file_ref, 'r') as h5:
            data_stub = h5.root.data[row0:row1]
            qual_stub = h5.root.quality[row0:row1]
            n_rows = len(h5.root.data)

>       data_fill = np.zeros(row0, dtype=data_stub.dtype)
E       ValueError: array is too big; `arr.size * arr.dtype.itemsize` is larger than the maximum possible size.

Ska/engarchive/tests/test_sync.py:192: ValueError
=============== 1 failed, 77 passed, 1 skipped in 261.64 seconds ===============
```

**Comment by taldcroft** [comment]
It is not expected per se, but this functionality is not supported on 32 bit and so the test should be skipped.

**Comment by jeanconn** [comment]
Sounds good.  If we care then, we can update the test runner to skip this on 32 bit.  For now I'll just ignore the test fail in the report.

**Comment by jeanconn** [comment]
Closed by #175
